neps/nep-0359.md

```diff
@@ -0,0 +1,168 @@
+
+## Unresolved Issues
+
+### Dealing with minted NTNFTs in cases of lost, stolen or new accounts
+
+Implementers should consider the case of what should happen to these NTNFTs if the user loses access to their account, it is somehow compromised or they simply wish to use a new account. As there are a number of ways this can be handled, this standard does not explicitly mandate that any particular function MUST be implemented to handle this.
+
+For each of these methods, it would presumably be necessary for the service which created the smart contract to make efforts (off-chain) to identify the user. Once they're satisfied the user has identified themselves they could use any of the following methods to invalidate the old NTNFT and, if required, mint a new token (presumably with the same metadata) to the new account.
+
+- The service would burn the NTNFT in the user's old account using: `function burn_token(token_id: string): null {}`. Giving the user access to this function may also be considered - in cases where they still have access to their account users could pre-emptively call this to prevent anyone using their NTNFTs to impersonate them.
+- The service could implement a `validity` status in the NTNFT's metadata that can be updated to show that the NTNFT is no longer valid. This would require the client services to understand the importance of this field and not simply check for the existence of the NTNFT when gating.
+
+The above methods may also be useful to services when they wish to revoke a users NTNFT, e.g. if the service finds the user was acting fraudulently.
+
+### Preventing transfer of account via trading wallet keys
+
+There will always exist the possibility that a user can simply transfer all the assets in their account, including any NTNFTs, to another user by simply giving them the private keys to the account.
+
+Ultimately, there is no way to prevent this. However, given common methods such as the one listed above for a user to do some sort of authenticated account recovery process, it would mean that any 'buyer' would run the risk the 'seller' simply recovers all their tokens using this process making such a trade less likely.
+
+## Future possibilities
+
+There are a few natural extensions to this standard which might be considered in NEPs for the future.
+
+The first is setting a standard for an 'authenticated account recovery' process. If such a process, like that described above, were standardised it would give users confidence that their NTNFTs would not be lost if they need to migrate to a new account. It would also enable frontend software to assist users in the account recovery process when they know the contract supports it.
+
+Other future NEPs might focus on more specific metadata which the NTNFT would provide, for specific use-cases. An example could be for when the NTNFT is used for KYC purposes that the metadata should specifically include fields such as `expiry date`, `validity` and `KYC_level`.
```

Comment:
The current standards actually account for that. There is e.g. an `expires_at` field, and specifically for domain-specific data (e.g. KYC), the `extra` field exists.

The existing `issued_at` and `expired_at` would also remove your need for a `validity` field, e.g. with a user authenticating himself every x months by depositing a yocto, and getting a valid new NTNFT, or the current being extended. I digress however, these details should be left to the implementer. Long story short: No need to extend the metadata.

Reply:
The new `validity` status was required for use cases where the smart contract owner might need to remove/update a status regardless of the `expired_at` date. 

Example 1: when a KYC provider issues a soulbound NFT as proof, the verified user faces certain charges, and regulators request to limit their ability to access financial services. The issuer would need to inform the relying services about this status change abruptly. 

Example 2: a DAO issued the soulbound NFT as a proof of membership; however, when implementing, they wanted the users to owe their proofs of "temporary" membership. However, the user might no longer be actively participating in the DAO. While `expired_at` could be used, it would give continuous work to the DAO operators, while with `validity`, the DAO can turn on/off the membership.

Reply:
One thing that confuses me in the KYC_level.  Know Your Customer level sounds funny.  In general I have found that KYC has morphed to mean identity verification so it's certainly not a misuse here, just find it strange.

Reply:
I agree with @willemneal that we shouldn't use terms like KYC. That's such a bizarre term / jargon that probably never should have been invented and certainly doesn't fit here.

Reply:
I think there's probably already an understanding that people will customize the metadata for their own uses without explicitly mentioning examples here in this standard. In an effort to keep this as simple as possible it might be better if we simply remove line 163 altogether then.

Reply:
I agree with @JasperTimm ^

Reply:
Removed the line regarding addn metadata in the latest commit